hail/python/hailtop/batch/backend.py

```diff
@@ -452,6 +452,26 @@ def __init__(self,
         gcs_kwargs = {'project': google_project}
         self.__fs: RouterAsyncFS = RouterAsyncFS(default_scheme='file', gcs_kwargs=gcs_kwargs)
 
+        self.regions: Optional[List[str]]
+        if isinstance(regions, str):
+            self.regions = [regions]
```

Comment:
I find it supremely confusing that a parameter which is plural can be both a single `str` and a list of str. Let's just make users write the `[xxx]`.